pkg/common/envvars.go

```diff
@@ -98,4 +98,13 @@ const (
 
 	// EnvAllowAutoRoundOffFilesystemSize specifies if auto round off minimum filesystem size is enabled
 	EnvAllowAutoRoundOffFilesystemSize = "CSI_AUTO_ROUND_OFF_FILESYSTEM_SIZE"
+
+	//EnvPodmonEnabled indicates that podmon is enabled
+	EnvPodmonEnabled = "X_CSI_PODMON_ENABLED"
+
+	//EnvPodmonAPIPORT indicates the port to be used for exposing podmon API health, TO-DO - Is it okay to rename this var that makes more sense. Ex: EnvPodmonArrayConnectivityAPIPORT
```

Comment:
Do we need to take care of this To-Do item mentioned here in the comment?

Reply:
Not for as of now. In case if other team will rename or will agree to rename, we will also do the same.